### PR TITLE
Add blog post for sprint 106 SCM/CI integration

### PR DESCRIPTION
**Pull request by eduardoj**

### CODE REVIEW

File: _includes/partials/_series-of-posts-about-scm-integration.md [hunk 1]
```diff
@@ -3,5 +3,6 @@ then [made some improvements](/2021/06/14/scm-integration-improvements/) in June
 We introduced advanced features like [reporting filters and support for self-hosted SCM](/2021/07/12/scm-integration-more-improvements)
 together with a list of [common pitfalls](/2021/07/12/scm-integration-pitfalls) in July 2021
 and in August 2021, we continued with [two new steps and a UI for tokens](/2021/08/02/link-package-configure-repositories-and-ui).
-After starting to [support more actions for pull/merge requests and improving the UI for tokens](/2021/09/14/scm-more-pull-request-actions-and-ui) in mid-September,
-we are now [adding support for push events and a rebuild step](/2021/09/28/support-for-push-events)._
+We started to [support more actions for pull/merge requests and improving the UI for tokens](/2021/09/14/scm-more-pull-request-actions-and-ui),
```
Comment (dmarcoux) [inline _includes/partials/_series-of-posts-about-scm-integration.md:6]:
```suggestion
After [supporting more actions for pull/merge requests and improving the UI for tokens](/2021/09/14/scm-more-pull-request-actions-and-ui),
```

File: _includes/partials/_series-of-posts-about-scm-integration.md [hunk 2]
```diff
@@ -3,5 +3,6 @@ then [made some improvements](/2021/06/14/scm-integration-improvements/) in June
...
+and [adding support for push events and a rebuild step](/2021/09/28/support-for-push-events) in September 2021.
+An now we present [user documentation and further improvements for the UI for tokens](/2021-11-09-scm-integration-documentation)._
```
Comment (dmarcoux) [inline _includes/partials/_series-of-posts-about-scm-integration.md:8]:
```suggestion
We now present [user documentation and further improvements for the UI for tokens](/2021-11-09-scm-integration-documentation)._
```

File: _posts/development/2021-11-09-scm-integration-documentation.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+more readable way by creating a new section in our OBS manual:
+[SCM/CI Workflow Integration](http://$URL_DOCU)
+
+Be aware that this is still a beta feature, so its documentation could change
+in the future. Please, have a look and keep your feedback coming.
+
+# Redesign of the Tokens UI
+
+The Tokens UI has been redesigned. This UI has not only improved in terms of
+resposiveness. It also allows you to name your tokens to make it easier to
```
Comment (dmarcoux) [inline _posts/development/2021-11-09-scm-integration-documentation.md:32]:
```suggestion
responsiveness, but it also allows you to name your tokens in order to easily identify them.
```

File: _posts/development/2021-11-09-scm-integration-documentation.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  <img src="/images/posts/.png" alt="List of tokens in the web UI" />
+  <figcaption>List of tokens in the web UI</figcaption>
+</figure>
+
+<figure>
+  <img src="/images/posts/.png" alt="Edit a token in the web UI" />
+  <figcaption>Edit a token in the web UI</figcaption>
+</figure>
+
+# ... and Much More
```
Comment (dmarcoux) [inline _posts/development/2021-11-09-scm-integration-documentation.md:47]: Starting a heading with dots is somewhat weird. How about we instead have:

```suggestion
# And Much More
```

**Comment by eduardoj** [comment]
@dmarcoux, thanks for your suggestions. I applied all of them. Please have a look again.

**Review by saraycp — APPROVED** [review]
Netlify hasn't deployed the new version correctly (yet). But fetching the branch and testing locally I can see the post fine. LGTM

**Comment by hennevogel** [comment]
rebased on master so things from the future are published.

**Pull request merged**